server.js

```diff
@@ -49,14 +49,17 @@ const validateHeaders = (headers) => {
     const lang_header = headers['accept-language-stt'].toLowerCase();
 
     // if the passed language contains anything different from two (eg. pt)
-    // or five (eg. pt-br) chars, we deny
-    if (lang_header.length !== 2 && lang_header.length !== 5) {
+    // or five (eg. pt-br) chars, or eleven (eg. cmn-Hans-CN)
```

Comment:
Are we sure these are the only valid lengths for expected values of this header?

Reply:
Yes. They are all here: https://github.com/mozilla/speech-proxy/blob/master/languages.json